File: app/src/main/java/com/sesame/noteproject/anim/AnimActivity.java
package com.sesame.noteproject.anim;

import android.animation.AnimatorSet;
import android.animation.ArgbEvaluator;
import android.animation.ObjectAnimator;
import android.animation.ValueAnimator;
import android.content.Context;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.sesame.noteproject.R;

public class AnimActivity extends AppCompatActivity {

    private TextView tv;

    public static void startActivity(Context context) {
        context.startActivity(new Intent(context, AnimActivity.class));
    }

    @Override
    protected void onCreate(@Nullable @org.jetbrains.annotations.Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_anim);
        tv = findViewById(R.id.tv);
    }

    public void onClick(View view) {
        tv.setTextColor(getResources().getColor(R.color.colorPrimary));
        ObjectAnimator textColorAnim = ObjectAnimator.ofArgb(tv, "textColor",
                getResources().getColor(R.color.colorPrimary),
                getResources().getColor(R.color.colorAccent),
                getResources().getColor(R.color.colorPrimary)
        );
//        textColorAnim.setEvaluator(new ArgbEvaluator());
        textColorAnim.setDuration(3000);
        textColorAnim.start();

//        ObjectAnimator alphaAnim = ObjectAnimator.ofFloat(tv, "alpha", 1f, 0f, 1f);
//        alphaAnim.setDuration(3000);
//        alphaAnim.start();

//        ObjectAnimator heightAnim = ObjectAnimator.ofInt(tv, "height", 100, 200, 100);
//        heightAnim.setDuration(3000).start();

//        ValueAnimator valueAnimator = ValueAnimator.ofInt(
//                getResources().getColor(R.color.colorAccent),
//                getResources().getColor(R.color.colorPrimary),
//                getResources().getColor(R.color.colorAccent)
//        );
//        valueAnimator.addUpdateListener(animation -> tv.setTextColor((Integer) animation.getAnimatedValue()));
//        valueAnimator.setDuration(3000);
//        valueAnimator.start();

//        AnimatorSet animatorSet = new AnimatorSet();
//        animatorSet.playTogether(
//                ObjectAnimator.ofFloat(tv, "alpha", 1f, 0f, 1f),
//                ObjectAnimator.ofInt(tv, "height", 100, 200, 100)
//        );
//        animatorSet.setDuration(3000);
//        animatorSet.start();
    }
}

File: module_test/src/main/java/com/sesame/module_test/test_anim/AnimActivity.kt
package com.sesame.module_test.test_anim

import android.os.Bundle
import android.view.animation.AlphaAnimation
import android.view.animation.LayoutAnimationController
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import com.sesame.module_test.R
import kotlinx.android.synthetic.main.activity_test_anim.*

class AnimActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_test_anim)

        val tvBlue = findViewById<TextView>(R.id.tvBlue)
        tvBlue.setOnClickListener {
//            val translateAnimation = AnimationUtils.loadAnimation(this, R.anim.anim_translate)
//            tvBlue.startAnimation(translateAnimation)

//            tvBlue.animate().rotation(360f).duration = 2000

//            val objectAnimator = ObjectAnimator.ofInt(tvValue, "value", 102, 923)
//            objectAnimator.duration = 5000
//            objectAnimator.interpolator = AccelerateInterpolator()
//            objectAnimator.start()

            layoutAnimation()
        }
    }

    private fun layoutAnimation() {
        val animation = AlphaAnimation(0f, 1f)
        animation.duration = 2000
        val controller = LayoutAnimationController(animation, 1f)
        controller.order = LayoutAnimationController.ORDER_NORMAL
        flContainer.layoutAnimation = controller

        for (index in 0..3) {
            val inflate = layoutInflater.inflate(R.layout.layout_animation, flContainer, false)
            flContainer.addView(inflate)
        }
    }
}